lib/Sources/App/AppModel.swift

```diff
@@ -62,8 +72,10 @@ public class AppModel {
                     urlFromClipboard = UIPasteboard.general.url
                     userHasBeenPromptedToPasteUrl = true
                 }
-
+                self.lastOpenedTabIndex = 1
                 self.activePageModel = buildPageModel(tabIndex: 1, initialUrl: urlFromClipboard)
+            } else if let lastOpenedTabIndex {
+                self.activePageModel = buildPageModel(tabIndex: lastOpenedTabIndex, initialUrl: tabsModel.urlForIndex(index: lastOpenedTabIndex))
```

Comment:
A small tweak I think would be preferrable to use the tab model:
```suggestion
            } else if let lastOpenedTabIndex, let tabModel = tabsModel.findTab(for: lastOpenedTabIndex) {
                self.activePageModel = buildPageModel(tabModel: tabModel)
```

Reply:
Didn't know tabsModel had that function. Nice.